lib/addons/adot/index.ts

```diff
@@ -17,7 +17,7 @@ const versionMap: Map<KubernetesVersion, string> = new Map([
 /**
  * Configuration options for the Adot add-on.
  */
-export type AdotCollectorAddOnProps = Omit<CoreAddOnProps, "saName" | "addOnName" > & {
+export type AdotCollectorAddOnProps = Omit<CoreAddOnProps, "saName" | "addOnName" | "version" > & {
```

Comment:
@Falco152 If we omit, we have to add version to `AdotCollectorAddOnProps`. Check `Cloud-Watch-Insights` Addon for example. Can you also make sure to test it once?